### PR TITLE
[13.0][MIG] mgmtsystem_environment

### PR DESCRIPTION
**Pull request by flachica**
@NachoAlesLopez @Guadaltech @max3903 Please, review

### CODE REVIEW

**Comment by Rad0van** [comment]
I guess this is waiting for #324 to be merged?

**Comment by flachica** [comment]
Yes, that is

**Comment by Rad0van** [comment]
> Yes, that is

Thanx - both for answer and for migrating. Hope it gets moving.

**Comment by flachica** [comment]
You can write here https://odoo-community.org/groups/contributors-15 if any volunteers is working on this

**Comment by flachica** [comment]
Or here https://odoo-community.org/groups/management-systems-17

**Comment by Rad0van** [comment]
But I guess someone just needs to let the merge happen on the other one to get this moving, right? So there is no real need for anyone to work on this, just some of the "moderators" approve it?

**Comment by flachica** [comment]
From my point of view, if we want the OCA modules to meet minimum quality standards, the moderator's job is not only to make the merge happen, they should also review the work done. I suggest you read the contributor manual. It explains perfectly what is the flow that is followed and solves any doubt of this type.

Another alternative is to fork the branches in the installation of your client and when the approval of the PR occurs, you change it.

These are my first contributions. Let's see if anyone sheds more light on this.

**Comment by Rad0van** [comment]
> From my point of view, if we want the OCA modules to meet minimum quality standards, the moderator's job is not only to make the merge happen, they should also review the work done. I suggest you read the contributor manual. It explains perfectly what is the flow that is followed and solves any doubt of this type.
> 
> Another alternative is to fork the branches in the installation of your client and when the approval of the PR occurs, you change it.
> 
> These are my first contributions. Let's see if anyone sheds more light on this.

I did and I thought I understood. But I misread the information about what's required: I thought it was  enough to get 2 positive reviews or 5 days :-) It requires both.

**Comment by Rad0van** [comment]
@flachica this could now go through. But don't know how to force Travis to attempt to rebuild.

**Comment by Rad0van** [comment]
@flachica sorry to bother but could you please force re-run of tests so this maybe passes? Thank you.

**Comment by flachica** [comment]
@Rad0van Forced

**Comment by Rad0van** [comment]
Could some of @NachoAlesLopez @guadaltech @max3903 @dreispt or @ivantodorovich review please? Thank you.

**Comment by Rad0van** [comment]
@flachica I found a strange behaviour. When I install mgmtsystem_environment and it installs document_page_environmental_aspect as its dependency all the documents and categories except for Environmental Aspect category dissapear. They are not deleted just cannot be seen. When you manually uninstall document_page_environmental_aspect (because in Management System configuration it is not checked as installed) all the documents and categories appear again.

File: mgmtsystem_environment/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Environment Management System",
+    "version": "13.0.1.0.0",
+    "author": "Savoir-faire Linux, Odoo Community Association (OCA)",
+    "website": "http://www.savoirfairelinux.com",
```
Comment (FernandoRomera) [inline mgmtsystem_environment/__manifest__.py]: "website": "https://github.com/OCA/management-system",

**Review by dreispt — APPROVED** [review]

**Comment by ivantodorovich** [comment]
@flachica there's just a quick pre-commit message to fix :)

**Review by ivantodorovich — APPROVED** [review]

**Comment by flachica** [comment]
@ivantodorovich pre-commit "now is green"

**Comment by ivantodorovich** [comment]
Thanks!!

/ocabot merge nobump

**Pull request merged**